cocos2d/CCScheduler.m

```diff
@@ -517,7 +693,107 @@ -(void)update:(CCTime)dt
 	[self updateTo:_currentTime + clampedDelta];
 	
 	InvokeMethods(_updates, @selector(update:), clampedDelta);
+    
+    #warning TODO: also invoke update actions:
```

Comment:
the line below invokes `updateActions` or does this warning refer to something else?